packages/block-editor/src/components/block-navigation/index.js

```diff
@@ -35,10 +36,18 @@ function BlockNavigationList( {
 				const blockType = getBlockType( block.name );
 				const isSelected = block.clientId === selectedBlockClientId;
 
+				/* HTML equivalent code
+				 * document.getElementsByClassName("editor-block-navigation__container")[0].closest(".components-popover__content").scrollTop
+				 *		= document.getElementsByClassName("components-button is-selected")[0].offsetTop
+				 */
+				const blockEl = useRef( null );
+				// selected_block.current.closest(".components-popover__content").scrollTop = block.offsetTop;
```

Comment:
I am unsure when to access the reference using `blockEl.current.closest(...)` as it did not work when I introduced a conditional here to attach a reference ONLY on `isSelected === true` case. Also, `blockEl` returned `null` here. Am I supposed to do it `onClick` of the `BlockNavigationDropdown` component?

Reply:
something like this should probably go inside a `useEffect` hook.